src/styles/main.scss

```diff
@@ -1 +1,72 @@
-// put code here
+* {
+  box-sizing: border-box;
+}
+
+.body {
+  align-items: center;
+  justify-content: center;
+}
+
+.clock {
+  margin: 100px 400px;
```

Comment:
The position of the clock should not be set with margin. It should be centered using display: flex; align-items: center; justify-content: center; on the body.